aerleon/lib/juniper.py

```diff
@@ -848,7 +856,7 @@ def _Comment(self, addr, exclude=False, line_length=132):
             logging.warning('Ignoring non IPv4 or IPv6 address: %s', addr)
         return rval
 
-    def _Group(self, group, lc=True):
+    def _Group(self, group: List[Union[int, Tuple[int, int], str]], lc: bool = True) -> str:
```

Comment:
Not necessarily in scope, but just spotted this duplicated expression on line 883:
```
if isinstance(el, str) or isinstance(el, str):
```

Reply:
done